examples/passport-login/src/models/user-identity.model.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class UserIdentity extends Entity {
+  @property({
+    type: 'string',
+    id: 1,
```

Comment:
why are we giving this id a value of 1?

Reply:
@emonddr I used it in place of true , they both seem to work alike. I will change it to true.

Reply:
agreed

Reply:
> @emonddr I used it in place of true , they both seem to work alike. I will change it to true.

Never seen this usage.